Android/app/src/main/java/app/intra/IntroDialog.java

```diff
@@ -58,26 +58,38 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle
       savedInstanceState) {
     View welcomeView = inflater.inflate(R.layout.intro_pager, container);
 
+    // Right-to-left locale support.
+    boolean rtl = false;
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
```

Comment:
Is it worth pulling this logic to a util class?

Reply:
Done.  PTAL.